backend/src/main/java/com/relay/model/statives/Stativ.java

```diff
@@ -6,20 +6,29 @@
 import javax.persistence.OneToMany;
 
 import com.relay.model.AbstractEntity;
+import com.relay.model.Relay;
 import com.relay.model.shelves.Shelve;
 
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
 
 @Entity
 @Data
+@RequiredArgsConstructor
 @EqualsAndHashCode(callSuper = true)
 public class Stativ extends AbstractEntity {
 
+    private final Integer number;
+
     /**
      * List of shelves
      */
     @OneToMany
     private List<Shelve> shelves;
 
+    public void addRelay(Relay relay) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)